cmd/mount.go

```diff
@@ -329,7 +330,22 @@ func prepareMp(newCfg *vfs.Config, mp string) (ignore bool) {
 		return
 	}
 
-	contents, err := os.ReadFile(path.Join(mp, ".config"))
+	prefixedConfigName := ".config"
+
+	entries, _ := os.ReadDir(mp)
```

Comment:
We should have a fast path for those don't have a special prefix

Reply:
Best we can do is try getting the Inode of `.config`:

```go
ino, _ := utils.GetFileInode(path.Join(mp, ".config"))
```

and check whether it is a special node.

We can't skip the loop altogether and just read the contents of `.config`, as that would defy the initial purpose of this PR (i.e., the case when the file `.config` exists and clashes with JuiceFS `.config`).

Reply:
Yes, we can check the inode of `.config`.